packages/payload/src/queues/localAPI.ts

```diff
@@ -80,4 +80,18 @@ export const getJobsLocalAPI = (payload: Payload) => ({
     })
     return result
   },
+
+  runByID: async (args: {
+    id: string
```

Comment:
This probably should be `id: string | number` to support PG without UUIDs unless we have a custom ID for jobs?